com.intellij.diagnostic.PluginException: Abandoned AnnotationBuilder - its 'create()' method was never called: Builder{message='null', myCurrentElement=LatexKeyValPairImpl(KEY_VAL_PAIR) (class nl.hannahsten.texifyidea.psi.impl.LatexKeyValPairImpl), myCurrentAnnotator=nl.hannahsten.texifyidea.highlighting.LatexAnnotator@24a4d3f1, severity=INFORMATION, range=(implicit)(1225,1235)} [Plugin: nl.rubensten.texifyidea]

### Type of JetBrains IDE (IntelliJ, PyCharm, etc.) and version
IntelliJ IDEA 2023.1.2 (build IU-231.9011.34)

### Operating System 
Mac OS X 13.4 (x86_64)

### TeXiFy IDEA version
0.7.30

### Description



### Stacktrace
```
com.intellij.diagnostic.PluginException: Abandoned AnnotationBuilder - its 'create()' method was never called: Builder{message='null', myCurrentElement=LatexKeyValPairImpl(KEY_VAL_PAIR) (class nl.hannahsten.texifyidea.psi.impl.LatexKeyValPairImpl), myCurrentAnnotator=nl.hannahsten.texifyidea.highlighting.LatexAnnotator@24a4d3f1, severity=INFORMATION, range=(implicit)(1225,1235)} [Plugin: nl.rubensten.texifyidea]
	at com.intellij.diagnostic.PluginProblemReporterImpl.createPluginExceptionByClass(PluginProblemReporterImpl.java:23)
	at com.intellij.diagnostic.PluginException.createByClass(PluginException.java:99)
	at com.intellij.codeInsight.daemon.impl.B.assertAnnotationCreated(B.java:333)
	at com.intellij.codeInsight.daemon.impl.AnnotationHolderImpl.assertAllAnnotationsCreated(AnnotationHolderImpl.java:265)
	at com.intellij.codeInsight.daemon.impl.DefaultHighlightVisitor.analyze(DefaultHighlightVisitor.java:93)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.analyzeByVisitors(GeneralHighlightingPass.java:311)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectHighlights(GeneralHighlightingPass.java:278)
	at com.intellij.codeInsight.daemon.impl.GeneralHighlightingPass.collectInformationWithProgress(GeneralHighlightingPass.java:226)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:97)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:57)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$1(PassExecutorService.java:382)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1102)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:374)
	at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeProcessUnderProgress$13(CoreProgressManager.java:604)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:679)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:635)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:603)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:60)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:373)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:349)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:229)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:187)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:347)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:181)
	at java.base/java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:373)
	at java.base/java.util.concurrent.ForkJoinPool$WorkQueue.topLevelExec(ForkJoinPool.java:1182)
	at java.base/java.util.concurrent.ForkJoinPool.scan(ForkJoinPool.java:1655)
	at java.base/java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1622)
	at java.base/java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:165)
Caused by: java.lang.IllegalStateException: Abandoned AnnotationBuilder - its 'create()' method was never called: Builder{message='null', myCurrentElement=LatexKeyValPairImpl(KEY_VAL_PAIR) (class nl.hannahsten.texifyidea.psi.impl.LatexKeyValPairImpl), myCurrentAnnotator=nl.hannahsten.texifyidea.highlighting.LatexAnnotator@24a4d3f1, severity=INFORMATION, range=(implicit)(1225,1235)}
	at com.intellij.codeInsight.daemon.impl.B.assertAnnotationCreated(B.java:332)
	... 26 more

```

--- a/src/nl/hannahsten/texifyidea/highlighting/LatexAnnotator.kt
+++ b/src/nl/hannahsten/texifyidea/highlighting/LatexAnnotator.kt
@@ -147,10 +147,10 @@ open class LatexAnnotator : Annotator {
     }
 
     private fun annotateKeyValuePair(element: LatexOptionalKeyValPair, annotationHolder: AnnotationHolder) {
-        element.keyValValue ?: return
+        val value = element.keyValValue ?: return
 
         annotationHolder.newSilentAnnotation(HighlightSeverity.INFORMATION)
-            .range(TextRange(element.optionalKeyValKey.endOffset, element.keyValValue!!.startOffset))
+            .range(TextRange(element.optionalKeyValKey.endOffset, value.startOffset))
             .textAttributes(LatexSyntaxHighlighter.SEPARATOR_EQUALS)
             .create()
     }

--- a/src/nl/hannahsten/texifyidea/highlighting/LatexAnnotator.kt
+++ b/src/nl/hannahsten/texifyidea/highlighting/LatexAnnotator.kt
@@ -11,9 +11,13 @@ import com.intellij.refactoring.suggested.endOffset
 import com.intellij.refactoring.suggested.startOffset
 import nl.hannahsten.texifyidea.index.LatexDefinitionIndex
 import nl.hannahsten.texifyidea.lang.Environment
+import nl.hannahsten.texifyidea.lang.commands.LatexGenericMathCommand.*
+import nl.hannahsten.texifyidea.lang.commands.LatexGenericRegularCommand
+import nl.hannahsten.texifyidea.lang.commands.LatexGenericRegularCommand.*
 import nl.hannahsten.texifyidea.psi.*
 import nl.hannahsten.texifyidea.util.labels.getLabelDefinitionCommands
 import nl.hannahsten.texifyidea.util.magic.CommandMagic
+import nl.hannahsten.texifyidea.util.magic.cmd
 import nl.hannahsten.texifyidea.util.parser.*
 
 /**
@@ -135,7 +139,7 @@ open class LatexAnnotator : Annotator {
                 .textAttributes(highlighter)
                 .create()
 
-            if (element.name == "\\text" || element.name == "\\intertext") {
+            if (element.name == TEXT.cmd || element.name == INTERTEXT.name) {
                 // Avoid creating an Annotation without calling the create() method
                 val range = element.requiredParameters().firstOrNull() ?: continue
                 annotationHolder.newSilentAnnotation(HighlightSeverity.INFORMATION)
@@ -226,14 +230,14 @@ open class LatexAnnotator : Annotator {
      */
     private fun annotateStyle(command: LatexCommands, annotationHolder: AnnotationHolder) {
         val style = when (command.name) {
-            "\\textbf" -> LatexSyntaxHighlighter.STYLE_BOLD
-            "\\textit" -> LatexSyntaxHighlighter.STYLE_ITALIC
-            "\\underline" -> LatexSyntaxHighlighter.STYLE_UNDERLINE
-            "\\sout" -> LatexSyntaxHighlighter.STYLE_STRIKETHROUGH
-            "\\textsc" -> LatexSyntaxHighlighter.STYLE_SMALL_CAPITALS
-            "\\overline" -> LatexSyntaxHighlighter.STYLE_OVERLINE
-            "\\texttt" -> LatexSyntaxHighlighter.STYLE_TYPEWRITER
-            "\\textsl" -> LatexSyntaxHighlighter.STYLE_SLANTED
+            TEXTBF.cmd -> LatexSyntaxHighlighter.STYLE_BOLD
+            TEXTIT.cmd -> LatexSyntaxHighlighter.STYLE_ITALIC
+            LatexGenericRegularCommand.UNDERLINE.cmd -> LatexSyntaxHighlighter.STYLE_UNDERLINE
+            SOUT.cmd -> LatexSyntaxHighlighter.STYLE_STRIKETHROUGH
+            TEXTSC.cmd -> LatexSyntaxHighlighter.STYLE_SMALL_CAPITALS
+            OVERLINE.cmd -> LatexSyntaxHighlighter.STYLE_OVERLINE
+            TEXTTT.cmd -> LatexSyntaxHighlighter.STYLE_TYPEWRITER
+            TEXTSL.cmd -> LatexSyntaxHighlighter.STYLE_SLANTED
             else -> return
         }
 

--- a/src/nl/hannahsten/texifyidea/lang/commands/LatexGenericMathCommand.kt
+++ b/src/nl/hannahsten/texifyidea/lang/commands/LatexGenericMathCommand.kt
@@ -111,6 +111,7 @@ enum class LatexGenericMathCommand(
     IMAGINARY("Im", display = "ℑ", collapse = true),
     DOTLESS_I("imath", display = "ı", collapse = true),
     INFINUM("inf"),
+    INTERTEXT("intertext", "text".asRequired(), dependency = LatexPackage.AMSMATH),
     INVERSE_F("Finv", dependency = AMSSYMB, display = "Ⅎ", collapse = true),
     INVERSED_NOT_SIGN("invneg", dependency = WASYSYM),
     DOTLESS_J("jmath", display = "ȷ", collapse = true),

--- a/src/nl/hannahsten/texifyidea/lang/commands/LatexGenericMathCommand.kt
+++ b/src/nl/hannahsten/texifyidea/lang/commands/LatexGenericMathCommand.kt
@@ -111,7 +111,7 @@ enum class LatexGenericMathCommand(
     IMAGINARY("Im", display = "ℑ", collapse = true),
     DOTLESS_I("imath", display = "ı", collapse = true),
     INFINUM("inf"),
-    INTERTEXT("intertext", "text".asRequired(), dependency = LatexPackage.AMSMATH),
+    INTERTEXT("intertext", "text".asRequired(), dependency = AMSMATH),
     INVERSE_F("Finv", dependency = AMSSYMB, display = "Ⅎ", collapse = true),
     INVERSED_NOT_SIGN("invneg", dependency = WASYSYM),
     DOTLESS_J("jmath", display = "ȷ", collapse = true),

